Makefile

```diff
@@ -1,10 +1,17 @@
-PROMBENCH_CMD        = ./prombench
-DOCKER_TAG = docker.io/prombench/prombench:2.0.0
+PROMBENCH_CMD := ./prombench
+DOCKER_REPO := prombench
+DOCKER_IMAGE_NAME := prombench
+DOCKER_IMAGE_TAG := 2.0.0
 
+include Makefile.common
+
+# Prombench Commands
 ifeq ($(AUTH_FILE),)
-AUTH_FILE = "/etc/serviceaccount/service-account.json"
+	AUTH_FILE = "/etc/serviceaccount/service-account.json"
 endif
 
+.PHONY: deploy clean
```

Comment:
(nit) I tend to have the `.PHONY` line next to the target to which it applies. In other words, `.PHONY: clean` should go right before the `clean: ...` definition.